external/ag-website-shared/src/components/example-runner/components/ExampleLogger.module.scss

```
@@ -35,6 +35,8 @@
     > div {
         padding: $spacing-size-2;
         border-bottom: 1px solid var(--color-border-secondary);
+        white-space: normal;
```

I think this should be pre-wrap. 

```suggestion
        white-space: pre-wrap;
```

---

Just in case people add extra spaces indentation ect

---

Updated